fluentui_core/src/main/java/com/microsoft/fluentui/util/DisplayUtils.kt

```diff
@@ -68,13 +68,6 @@ val Context.desiredDialogSize: IntArray
         return dialogSize
     }
 
-// This check returns false in emulators
-val Context.hasSoftNavigationBar: Boolean
```

Comment:
I found the check faulty on Pixel 5 emulator as well as physical device. As we get 0 height of navbar in case it is missing, removing this check should be beneficial.